gulpfile.js

```diff
@@ -289,10 +289,12 @@ const criticalCssConfig = (function () {
         return {
             article: global.concat(
                 /\.content-header__item_toggle--.*$/,
+                /\.info-bar.*/,
```

Comment:
`\.info-bar((--|__).+)?$` is technically safer (in case we ever end up with `.info-bars` or something).